### PR TITLE
[ready-to-merge] Document deleteSpaceTemplate mutation

### PR DESCRIPTION
**Pull request by a-kent**
@andrewegan Initial review?

### CODE REVIEW

File: guides/V1_delete_space_template.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+A user can delete a custom space template with the _deleteSpaceTemplate_ mutation.
+
+## Schema
+
+### Delete Space Template Mutation
+
+```graphql
+type MutationRoot {
+  ...
+  deleteSpaceTemplate(input: DeleteSpaceTemplateInput!): DeleteSpaceTemplateMutation!
```
Comment (andrewegan) [inline guides/V1_delete_space_template.md]: Is this non-null correct?

Comment (a-kent) [reply guides/V1_delete_space_template.md]: Nope; fixed.

**Review by andrewegan — APPROVED** [review]

**Comment by a-kent** [comment]
@jarusso @miguelestrada Ready for your review.

**Comment by jarusso** [comment]
looks good to me

**Pull request merged**